CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## Next release
 
+## 0.32.0 / 2019-08-30
+
+* [CHANGE] Change PodManagement policy to parallel in Alertmanager (#2676)
+* [CHANGE] Add golangci-lint and promtool to jsonnet-ci (#2739)
```

Comment:
Only user facing changes go into the changelog, this is a tooling change. It doesn't need to be noted here.

Reply:
Done.